src/cli.rs

```diff
@@ -243,5 +255,10 @@ pub fn extract_args(matches: &ArgMatches) -> Config {
     args.txs_batch_size = matches
         .value_of("batch-size")
         .map(|batch_size_str| batch_size_str.parse().expect("can't parse batch-size"));
+
+    args.priority_fees_proba = match matches.value_of("prioritization-fees") {
+        Some(x) => x.parse().expect("Prioritization fees is between 1-100"),
```

Comment:
Looks like it doesn't check the range.
But I think you can add a validator (comment above)

Reply:
Yes i will add a validator comment above.